flutter_bird_app/lib/controller/authentication_service.dart

```diff
@@ -103,37 +151,70 @@ class AuthenticationServiceImpl implements AuthenticationService {
 
         Uri? uri = resp.uri;
         if (uri != null) {
-          if (kIsWeb) {
-            webQrData = uri.toString();
+          // Web
+          if (kIsWeb && !isInLiff) {
```

Comment:
Will this condition also be applied when searching the web on mobile?
If so, how about launching a wallet for every mobile?

Reply:
Yes, what I meant was to add a check for mobile. 

> If I want to separate the behavior for PC browsers and mobile browsers, I will need a different check. So I would like to make that a separate issue. What do you think?

That sounds good to me.

Reply:
Created: https://github.com/Finschia/flutter-bird/issues/20